assignment/index.html

```diff
@@ -3,11 +3,16 @@
   <head>
     <meta charset="utf-8">
     <title>Unit 5, Part 4 - Music Collection</title>
-    <script src="scripts/3-music-collection.js" charset="utf-8"></script>
   </head>
   <body>
     <h1>Unit 5, Part 4 - Music Collection</h1>
     <p>See console for output.</p>
     <!-- Create an unordered list with any questions you have -->
+    <ul>
+      <li>This is random, but if I needed a quick refresher on semicolons, where could I go?</li>
+      <li>I believe we only need them after single-line declarations (variables, arrow functions...)</li>
+      <li>And not after loops - correct?</li>
```

Comment:
Thanks alot Zac - really appreciate the examples here - that helps me visualize things rather than thinking through it conceptually which I think will greatly benefit my retention. And makes sense on the practice/repetition!